projects/storefrontlib/src/cms-structure/outlet/outlet.service.ts

```diff
@@ -94,9 +94,12 @@ export class OutletService<T = TemplateRef<any> | ComponentFactory<any>> {
     } else if (value && store.has(outlet)) {
       let existing = store.get(outlet);
 
-      existing = existing.filter((val) => val !== value);
+      // Compare and remove specified selector
+      existing = existing?.filter(
+        (val) => (val as any).selector !== (value as any).selector
```

Comment:
Do you have a confirmation that this one will be not a breaking change?

Reply:
This will be taken from `develop` soon
https://github.com/SAP/spartacus/pull/13635